todo_app/templates/index.html

```diff
@@ -1,18 +1,34 @@
 {% extends "layout.html" %}
-{% block title %}To-Do App{% endblock %}
-
+{% block title %}Trello Board{% endblock %}
 {% block content %}
   <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <h1 class="display-5">To-Do Board</h1>
+    <p class="lead">Cards</p>
   </div>
-
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h5>To-Do</h5>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for itm in items %} 
+          <li>{{ itm.id }} - {{itm.name}} - {{itm.desc}}</li> 
+        {% endfor %} 
       </ul>
+      <h5>Doing</h5>
+      <ul class="list-group mb-4">
+        {% for doitm in doingitems %} 
+          <li>{{ doitm.id }} - {{doitm.name}} - {{doitm.desc}}</li> 
```

Comment:
You could have a `<form>` for each item with buttons to move them between lists, that way you wouldn't have to copy-paste the ids into the move card form.